gen_graphs.py

```diff
@@ -62,7 +62,7 @@ def gen_cdf_files() -> list[tuple[str, str, int]]:
             f.write(".headers off\n")
             f.write(".mode csv\n")
             f.write(".output "+fname_csv+"\n")
-            f.write("select t from results where solver='"+solver+"'\n and result!='unknown'")
+            f.write("select t from results where solver='"+solver+"'\n and result!='unknown' and correct!=false")
```

Comment:
```suggestion
            f.write(f"select t from results where solver='{solver}'\n and result!='unknown' and correct!=false")
```

might be useful to create an enum for possible result values rather than hardcoding `'unknown'` here

Reply:
Yeah, good point about the `unknown`... I'll do that sometime!